apps/ledger-live-desktop/src/renderer/modals/Platform/Exchange/CompleteExchange/TransactionBroadcastedContent.tsx

```diff
@@ -30,24 +32,32 @@ export function TransactionBroadcastedContent(props: TransactionBroadcastedConte
         provider={provider}
         {...swapDefaultTrack}
```

Comment:
I do not think have sense track swap properties here.

Reply:
Good catch, will create a hook for sell properties.